stingray/lightcurve.py

```diff
@@ -43,6 +55,12 @@ def __init__(self, time, counts, input_counts=True, gti=None):
             They will be used by other methods to have an indication of the
             "safe" time intervals to use during analysis.
 
+        statistic: str, optional, default=None
+            Statistic of the Lightcurve, it is used to calculate the
+            uncertainties and other statistical values apropriately.
+            Default makes no assumptions and keep errors equal to zero.
+
```

Comment:
Could we refactor this attribute? For various reasons, I don't like the use of `statistic` to mean type of uncertainties. I'm worried this could be really confusing. Maybe `err_dist` (for "distribution of uncertainties") or something like that? 

This should probably also contain a list of currently accepted options for `statistic`?  So that people know what they can put in?